tensorboard/BUILD

```diff
@@ -15,6 +16,13 @@ package_group(
     packages = ["//tensorboard/..."],
 )
 
+# The standard protos used in TensorBoard.
+tb_proto_library(
+    name = "protos_all",
+    srcs = glob(["proto/*.proto"]),
```

Comment:
Conventionally better for this to be in a BUILD file in the proto directory itself, rather than using a subdirectory glob.

Reply:
I decided to not use glob at all (for transparency) and moved it to `tensorboard/compat/proto/`